src/gui/spaces/spacesmodel.cpp

```diff
@@ -107,8 +107,8 @@ QVariant SpacesModel::data(const QModelIndex &index, int role) const
     case Qt::DecorationRole:
         switch (column) {
         case Columns::Image: {
-            auto it = _images.find(item.getId());
-            if (it != _images.cend()) {
+            auto it = _images.constFind(item.getId());
```

Comment:
hmm style question, should we stick to the std style named iterators?

Reply:
Using `constFind()` (and friends) ensures that we don't do a detach on the `_images` map. And being explicit about it helps to show that.

Reply:
hm yes but that would mean `constFind()` + `cend()` ?